hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/rollout/rollout/AddUpdateRolloutWindowLayout.java

```diff
@@ -691,15 +668,18 @@ public void validate(final Object value) {
         @Override
         public void validate(final Object value) {
             try {
-                new RegexpValidator(NUMBER_REGEXP, i18n.get(MESSAGE_ENTER_NUMBER)).validate(value);
-                new IntegerRangeValidator(i18n.get(MESSAGE_ROLLOUT_FIELD_VALUE_RANGE, 0, 500), 0, 500)
-                        .validate(Integer.valueOf(value.toString()));
+                if (value != null) {
+                    new IntegerRangeValidator(i18n.get(MESSAGE_ROLLOUT_FIELD_VALUE_RANGE, 0, 500), 0, 500)
+                            .validate(Integer.valueOf(value.toString()));
+                }
             }
             // suppress the need of preserve original exception, will blow
             // up the
             // log and not necessary here
             catch (@SuppressWarnings("squid:S1166") final InvalidValueException ex) {
-                LOG.error(ex.getMessage());
+                // we have to throw the exception here, otherwise the UI won't
```

Comment:
You do not suppress the exception anymore. So you can remove the `@SuppressWarnings`